moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -58,27 +73,25 @@ export const ProjectsListViewQueryConf = {
     date_added: {
       searchable: false,
       sortable: true,
-      label_search: null,
-      label_table: "Date Added",
+      label: "Date Added",
       width: "10%",
       filter: value => new Date(value).toLocaleDateString(),
       type: "date_iso",
     },
     start_date: {
       searchable: false,
       sortable: true,
-      label_search: null,
-      label_table: "Start Date",
+      label: "Start Date",
       width: "10%",
       filter: value => new Date(value).toLocaleDateString(),
```

Comment:
I'm not sure if this is where the bug is coming from, but I noticed a difference between the `start_date` returned from Hasura and the one displayed in the UI. The filter for `start_date` works as expected so it is just the formatting of this field in the UI that is affected.

<img width="750" alt="Screen Shot 2021-01-07 at 10 12 55 AM" src="https://user-images.githubusercontent.com/37249039/103934781-2e582f80-50eb-11eb-9ba6-0270576a3c7a.png">

Reply:
Thanks! I need to investigate this, I think kickstand is kicking on me. Locally it works, but I know for a fact I have the latest GQLAbstract, which does the filtering.

It could be that the data in staging is different than the data locally.  I have a couple leads. 🙏

Reply:
This is what it looks like locally:
![2021-01-07_14-24-51](https://user-images.githubusercontent.com/5282430/103941577-af67f480-50f4-11eb-92f0-4892cf01e5a2.png)